ci/build-docker-image.sh

```diff
@@ -10,12 +10,11 @@ set -o errexit
 # Execute script only in a CI environment
 if [ "$CI" != "true" ]
 then
-    echo -e "\e[35m\e[1mError!\e[0m Trying to execute a CI script in a non-CI environment. Exiting script."
+    echo -e "\e[35;1mError!\e[0m Trying to execute a CI script in a non-CI environment. Exiting script."
```

Comment:
@MatthijsBurgh can you have a look at only this file and tell me what do you feel about the changes?